.env.example

```diff
@@ -0,0 +1,13 @@
+COGNITE_TOKEN_URL=
+COGNITE_CLIENT_ID=
+COGNITE_CLIENT_SECRET=
+COGNITE_TOKEN_SCOPES=
+COGNITE_CLIENT_NAME=
+COGNITE_PROJECT=
+COGNITE_BASE_URL=
+COGNITE_STATE_DB=
+COGNITE_STATE_TABLE=
+COGNITE_EXTRACTION_PIPELINE=
+LAKEHOUSE_BASE_URL=
```

Comment:
Line 11 and 13 doesn't seem to be needed anymore given the lakehouse config is no longer used.